### ISSUE TITLE
Frogs clip through the floor and into the endless void

### ISSUE THREAD

**Issue by Magnogen** [l2wilson94/TimePond#1]
When placing a portal at the bottom of a world, a froggo can find itself in the endless void of eternal nothingness, acompanied only by pain, suffering and anguish - never to be seen or heard from again. The frog undergoes complete and utter inter-thermal damnation within the eternal void and its suffering will not be relieved even by means of a merciful death.

Save froggo from clipping into the barrier that seperates the comfortable life the froggo knows, loves and cherishes, from the everlasting void of nothingness and despair - for they are only to be greeted with eternal torment.

![screenshot](https://user-images.githubusercontent.com/25611707/141151821-ab815643-40c1-4aa2-a1c7-2e3adc595620.png)

tl;dr blocks can touch the floor on the otherside of a portal, but a frog will keep going

**Comment by Magnogen** [comment]
Happy Froggo :D

![Happy Froggo](https://user-images.githubusercontent.com/25611707/141156780-43859878-ee18-40bf-aa0a-d04146cd6bb0.gif)

**Comment by l2wilson94** [comment]
Awesome! Also there are loooads of URL parameters that you can use to try the different experiments out. Such as https://www.timepond.cool/?experiment=gfling&menu=purple&portal=pastnow

**Comment by macxwdc** [comment]
can there be a list showing each type of experiment?

**Comment by TodePond** [comment]
Hey, I added some presets to the readme. Here are some you can try:


|  | Simple | Right | Fling | Fall | Bump |
| --- | --- | --- | --- | --- | --- |
| Teleport | [Teleport + Simple](https://timepond.cool?experiment=gsimple&menu=cyan&portal=move) | [Teleport + Right](https://timepond.cool?experiment=gright&menu=cyan&portal=move) | [Teleport + Fling](https://timepond.cool?experiment=gfling&menu=cyan&portal=move) | [Teleport + Fall](https://timepond.cool?experiment=gfall&menu=cyan&portal=move) | [Teleport + Bump](https://timepond.cool?experiment=ggen&menu=cyan&portal=move) | 
| Future | [Future + Simple](https://timepond.cool?experiment=gsimple&menu=cyan&portal=futurenow) | [Future + Right](https://timepond.cool?experiment=gright&menu=cyan&portal=futurenow) | [Future + Fling](https://timepond.cool?experiment=gfling&menu=cyan&portal=futurenow) | [Future + Fall](https://timepond.cool?experiment=gfall&menu=cyan&portal=futurenow) | [Future + Bump](https://timepond.cool?experiment=ggen&menu=cyan&portal=futurenow) | 
| Past | [Past + Simple](https://timepond.cool?experiment=gsimple&menu=cyan&portal=pastnow) | [Past + Right](https://timepond.cool?experiment=gright&menu=cyan&portal=pastnow) | [Past + Fling](https://timepond.cool?experiment=gfling&menu=cyan&portal=pastnow) | [Past + Fall](https://timepond.cool?experiment=gfall&menu=cyan&portal=pastnow) | [Past + Bump](https://timepond.cool?experiment=ggen&menu=cyan&portal=pastnow) |
| Rewrite | [Rewrite + Simple](https://timepond.cool?experiment=gsimple&menu=purple&portal=rewrite) | [Rewrite + Right](https://timepond.cool?experiment=gright&menu=purple&portal=rewrite) | [Rewrite + Fling](https://timepond.cool?experiment=gfling&menu=purple&portal=rewrite) | [Rewrite + Fall](https://timepond.cool?experiment=gfall&menu=purple&portal=rewrite) | [Rewrite + Bump](https://timepond.cool?experiment=ggen&menu=purple&portal=rewrite) |
| Branch | [Branch + Simple](https://timepond.cool?experiment=gsimple&menu=purple&portal=pastline) | [Branch + Right](https://timepond.cool?experiment=gright&menu=purple&portal=pastline) | [Branch + Fling](https://timepond.cool?experiment=gfling&menu=purple&portal=pastline) | [Branch + Fall](https://timepond.cool?experiment=gfall&menu=purple&portal=pastline) | [Branch + Bump](https://timepond.cool?experiment=ggen&menu=purple&portal=pastline) |

I left some of the other presets in a broken state, (eg: Nexus) so I'll fix those at some point and add them too!
